[BLOCKER] RAILWAY_TOKEN_ACC1/ACC2 secrets invalid — writer-env-fix cannot heal writer

## Problem

Cure A (`writer-env-fix.yml`) failed on run #2 (2026-05-14T05:35Z) because **RAILWAY_TOKEN_ACC1 secret is invalid**. Error:

```
##[error]token ACC1 failed sanity check
##[error]Process completed with exit code 3
```

Backboard GraphQL endpoint `https://backboard.railway.com/graphql/v2` returned non-success when queried with this token — meaning Railway has revoked or rotated the token, but the GitHub Actions secret was not updated.

## Impact

- IGLA writer dead 560+ minutes (`ssot.bpb_samples` stalled since 2026-05-13 20:14Z).
- Cure A cannot run.
- All matrix-runner deploys are unreachable through this token path.

## Operator action required

1. Generate fresh **team-scoped** Railway tokens for ACC1 + ACC2 in https://railway.app/account/tokens.
2. Update GitHub secrets in `gHashTag/trios-railway`:
   - `RAILWAY_TOKEN_ACC1`
   - `RAILWAY_TOKEN_ACC2`
3. Re-run `writer-env-fix.yml` with `confirm=PHI`.
4. Verify via SQL probe:
   ```sql
   SELECT COUNT(*) FILTER (WHERE ts > now() - interval '5 minutes')::int AS rows_5m,
          ROUND(EXTRACT(EPOCH FROM (now() - MAX(ts)))/60) AS min_since_last
   FROM ssot.bpb_samples WHERE canon_name LIKE 'IGLA-%';
   ```
   Expect `rows_5m > 0` within 5 min.

## Evidence

Failed run: https://github.com/gHashTag/trios-railway/actions/runs/25843795652
Throne meta: gHashTag/trios#264

Anchor: phi^2 + phi^-2 = 3

--- a/.github/workflows/writer-env-fix.yml
+++ b/.github/workflows/writer-env-fix.yml
@@ -1,4 +1,4 @@
-name: Writer Env Fix (NEON_DATABASE_URL on matrix runners)
+name: Writer Env Fix (DATABASE_URL on matrix runners)
 
 # Anchor: phi^2 + phi^-2 = 3 · DOI 10.5281/zenodo.19227877
 #
@@ -275,7 +275,7 @@ jobs:
             SVC_ID="${ID_PREFIX%%-0000-0000-0000-000000000000}"
             echo "::group::IGLA $NAME ($SVC_ID)"
             ok=1
-            upsert_var "$TOKEN_ACC1" "$IGLA_PROJECT_ID" "$IGLA_ENV_ID" "$SVC_ID" "NEON_DATABASE_URL" "$RW_REF" || ok=0
+            upsert_var "$TOKEN_ACC1" "$IGLA_PROJECT_ID" "$IGLA_ENV_ID" "$SVC_ID" "DATABASE_URL" "$RW_REF" || ok=0
             redeploy   "$TOKEN_ACC1" "$SVC_ID" "$IGLA_ENV_ID" || ok=0
             if [[ "$ok" -eq 1 ]]; then
               echo "[OK] $NAME $SVC_ID"
@@ -307,7 +307,7 @@ jobs:
             SVC_ID="${ID_PREFIX%%-0000-0000-0000-000000000000}"
             echo "::group::acc2 $NAME ($SVC_ID)"
             ok=1
-            upsert_var "$TOKEN_ACC2" "$ACC2_PROJECT_ID" "$ACC2_ENV_ID" "$SVC_ID" "NEON_DATABASE_URL" "$RW_REF" || ok=0
+            upsert_var "$TOKEN_ACC2" "$ACC2_PROJECT_ID" "$ACC2_ENV_ID" "$SVC_ID" "DATABASE_URL" "$RW_REF" || ok=0
             redeploy   "$TOKEN_ACC2" "$SVC_ID" "$ACC2_ENV_ID" || ok=0
             if [[ "$ok" -eq 1 ]]; then
               echo "[OK] $NAME $SVC_ID"
@@ -324,7 +324,7 @@ jobs:
           # ===================================================================
           echo "::group::IGLA trios-railway-mcp ($MCP_SERVICE_ID) — last on purpose"
           ok=1
-          upsert_var "$TOKEN_ACC1" "$IGLA_PROJECT_ID" "$IGLA_ENV_ID" "$MCP_SERVICE_ID" "NEON_DATABASE_URL"    "$RW_REF" || ok=0
+          upsert_var "$TOKEN_ACC1" "$IGLA_PROJECT_ID" "$IGLA_ENV_ID" "$MCP_SERVICE_ID" "DATABASE_URL"         "$RW_REF" || ok=0
           upsert_var "$TOKEN_ACC1" "$IGLA_PROJECT_ID" "$IGLA_ENV_ID" "$MCP_SERVICE_ID" "RAILWAY_POSTGRES_URL" "$RW_REF" || ok=0
           redeploy   "$TOKEN_ACC1" "$MCP_SERVICE_ID" "$IGLA_ENV_ID" || ok=0
           if [[ "$ok" -eq 1 ]]; then

